assets/blocks/learner-courses-block/learner-courses-edit.js

```diff
@@ -17,6 +17,55 @@ import { __ } from '@wordpress/i18n';
 import CourseProgress from '../../shared/blocks/course-progress';
 import LearnerCoursesSettings from './learner-courses-settings';
 
+/**
+ * Featured image placeholder element.
+ */
+const FeaturedImagePlaceholder = () => (
+	<div
+		className="wp-block-sensei-lms-learner-courses__courses-list__featured-image"
+		role="img"
+		aria-label="Featured image"
+	>
+		<Icon icon={ image } size={ 48 } />
+	</div>
+);
+
+/**
+ * Wrapper for CSS variables & related classes.
+ *
+ * @param {Object} props
+ * @param {string} props.tag       HTML tag.
+ * @param {Array}  props.variables CSS variables.
+ * @param {Array}  props.children  Children elements.
+ * @param {*}      props.className Classes.
+ */
+const StylesWrapper = ( { tag, variables, children, className } ) => {
+	const isEmpty = ( value ) => {
+		return [ undefined, null, 'undefinedpx' ].includes( value );
+	};
+	const Tag = tag || 'div';
+	return (
+		<Tag
+			className={ classnames( className, {
+				'has-sensei-primary-color': !! variables.primaryColor,
+				'has-sensei-accent-color': !! variables.accentColor,
```

Comment:
The CSS variables can't fall back to a color set by another rule (for example, to the default link color for course titles). So rules using these variables are only applied when the variables are set, like this:

```css
.has-sensei-primary-color {
	.course-title a {
		color: var(--sensei-primary-color, inherit);
	}
```